src/Evolution/Systems/GrMhd/ValenciaDivClean/Observe.hpp

```diff
@@ -45,10 +45,6 @@ using reduction_datums =
 
 /*!
  * \brief Temporary action for observing volume and reduction data
```

Comment:
@nilsdeppe Are these actions still temporary or have they become what we want in the long term?

Reply:
Still temporary

Reply:
Thanks for confirming; I wanted check while I was touching the nearby code already :+1: